glitter/pages/models.py

```diff
@@ -43,15 +45,25 @@ class Meta(GlitterMixin.Meta):
     def get_absolute_url(self):
         return self.url
 
+    def get_latest_version(self):
```

Comment:
Let's put this in `GlitterMixin` instead? Seems like a nicer place for it as it can be reused.